### ISSUE TITLE
visualize button is hard to find

### ISSUE THREAD

**Issue by volodymyrss** [esg-epfl-apc/contributed-galaxy-workflows#143]
![image](https://github.com/user-attachments/assets/cae56047-7a92-4822-9965-d75be816f05b)

**Comment by bgruening** [comment]
Yes, there is an issue somewhere that we want to use the eye symbol for that as well. However, I think we need Vue3 at first. @dannon might know more.
